doc/Customization-guide/Endpoint-SDKs/C-SDK/Porting-guide/index.md

```diff
@@ -4,3 +4,69 @@ title: Porting guide
 permalink: /:path/
 sort_idx: 40
 ---
+
+{% include variables.md %}
+
+* TOC
+{:toc}
+
+
+# Prerequisites
+
+Kaa C SDK can be ported to all devices  with or without any operations sytem. For example it can be ported to: Linux, Windows, Android, IOS, FireFox, Solaris, microcontrollers with or without any Rtos, etc.
+Kaa supports Big and Little Endian platforms automaticaly, user doesn't need to care about it.
+Note! A platform must have enough memory for the SDK and a network controller with TCP-IP stack.
+
+# Dependencies
+
+The SDK uses a platform-level layer that depends on the environment where the SDK should run. 
+If you want to port it, you must modified a few files of the SDC and implement own functios. These files provide:
+
+## Mandatory		 	
+	
+ - Dynamic allocation routines of the memory, [interface and implementation]({{github_url}}client/client-multi/client-c/src/kaa/platform-impl/esp8266/platform/mem.h). 
```

Comment:
"Dynamic allocation routines of the memory" - The routines for dynamic memory allocation.
"interface and implementation" - there is no implementation.